MANIFEST.in

```diff
@@ -4,3 +4,4 @@ include src/poke_env/data/static/*.html
 include src/poke_env/data/static/moves/*.json
 include src/poke_env/data/static/pokedex/*.json
 include src/poke_env/data/static/typechart/*.json
+include src/poke_env/py.typed
```

Comment:
What's the point of this file?

Reply:
That file essentially tells pip that the poke_env package now has stub files built into it, and so this prevents the error that I mentioned in #393, which is the following: `Stub file not found for "poke_env"`.